Makefile

```diff
@@ -173,6 +175,9 @@ test/addons/.buildstamp: deps/npm/node_modules/node-gyp/package.json \
 # TODO(bnoordhuis) Force rebuild after gyp update.
 build-addons: $(NODE_EXE) test/addons/.buildstamp
 
+clean-addons:
```

Comment:
You will probably want to add this to the `.PHONY` target at the bottom of the Makefile